### PR TITLE
💄Meetings Page

### PR DESCRIPTION
**Pull request by joshbermanssw**
As per https://github.com/tinacms/tina.io/issues/1876 i have created a /meetings page as per our design spec. This page is helpful as it gives a direct link for potential clients to book via (instead of the modal on the homepage)

_test passed by @landonmaxwell_

Screenshot 2024-07-08 at 4 31 40 PM
**Figure: LG view **

Screenshot 2024-07-08 at 4 32 16 PM
**Figure: Tablet View**

### CODE REVIEW

File: components/blocks/Booking.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            <h3 className="w-full text-center mb-6 inline-block m-0 pb-4 text-lg md:whitespace-nowrap lg:leading-tight text-black">
+              Book a meeting with a 🦙TinaCMS expert in your timezone{' '}
+            </h3>
+            {meetingPeople.map((cardItem, idx) => (
+              <BookingCard key={idx} cardItem={cardItem} />
+            ))}
+          </div>
+        </div>
+      </div>
+    </div>
```
Comment (bradystroud) [inline components/blocks/Booking.tsx:100]: This is a lot of divs, can you try clean it up a bit?
Also not sure where this is actually used I don't remember seeing it anywhere

File: components/blocks/Booking.template.ts
Comment (bradystroud) [inline components/blocks/Booking.template.ts:1]: this is a duplicate of what is in the schema.
Try and find a way to remove the duplication

I think you might be able to use this template in the schema (see my comment on that file)

File: tina/schema.tsx [hunk 2]
```diff
@@ -481,5 +482,38 @@ export const schema = defineSchema({
         },
       ],
     },
+    {
+      name: 'meetingLinks',
+      label: 'Meeting Links',
+      path: 'content/meeting-links',
+      format: 'json',
+      fields: [
```
Comment (bradystroud) [inline tina/schema.tsx:490]: I think you can do something like this to remove the duplicated fields

```suggestion
      template: bookingTemplate 
      fields: [
```

File: tina/schema.tsx [hunk 3]
```diff
@@ -481,5 +482,38 @@ export const schema = defineSchema({
...
+              ui: {
+                itemProps: (item) => ({
+                  label: item.name,
+                }),
+              },
+              fields: [
+                { name: 'name', label: 'Name', type: 'string' },
+                { name: 'description', label: 'Description', type: 'string' },
+                { name: 'image', label: 'Image', type: 'image', description: 'Image headshot for a meeting card' },
+                { name: 'url', label: 'URL', type: 'string' },
```
Comment (bradystroud) [inline tina/schema.tsx:511]: Add some info about what kind of URL (hubspot meeting booking link

(Tip: Always assume your editors are silly geese, give them lots of tips).

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: tina/schema.tsx (hunk 3 above)
Comment (joshbermanssw) [reply tina/schema.tsx:511]: 🪿

File: components/blocks/Booking.tsx (hunk 1 above)
Comment (joshbermanssw) [reply components/blocks/Booking.tsx:100]: Lines 1-86 are the Step Indicator at the top of the page
- each "<div className="step-icon w-6 h-6 border-2 border-orange-300 flex items-center justify-center rounded-full"></div>" is the circle 

- each <div className="indicator-line h-0.5 w-20 md:w-40 bg-gray-300"></div> is the line between each circle

Comment (joshbermanssw) [reply components/blocks/Booking.tsx]: ✅ though

File: components/blocks/Booking.template.ts
Comment (joshbermanssw) [reply components/blocks/Booking.template.ts:1]: Addressed

File: tina/schema.tsx (hunk 2 above)
Comment (bradystroud) [reply tina/schema.tsx:490]: As per our conversation, fixed differently

**Review by bradystroud — APPROVED** [review]
LGtm

**Pull request merged**